modules/components/pages/outputs/kafka_franz.adoc

```diff
@@ -10,7 +10,9 @@
 component_type_dropdown::[]
 
 
-A Kafka output using the https://github.com/twmb/franz-go[Franz Kafka client library^].
+The `kafka_franz` output writes a batch of messages to Kafka brokers and waits for acknowledgement before propagating them back to the input. This output often outperforms the traditional `kafka` output, as well as providing more useful logs and error messages.
```

Comment:
Same as above. reworded as it's not clear what is being written back to the input.

Reply:
Updated text to make this clearer.